File: FE-Cinema/src/app/entity/userNoAccountDTO.ts
export class UserNoAccountDTO {
  private _name: string;
  private _email: string;
  private _idCard: string;
  private _phone: string;

  constructor(name: string, email: string, idCard: string, phone: string) {
    this._name = name;
    this._email = email;
    this._idCard = idCard;
    this._phone = phone;
  }


  get name(): string {
    return this._name;
  }

  set name(value: string) {
    this._name = value;
  }

  get email(): string {
    return this._email;
  }

  set email(value: string) {
    this._email = value;
  }

  get idCard(): string {
    return this._idCard;
  }

  set idCard(value: string) {
    this._idCard = value;
  }

  get phone(): string {
    return this._phone;
  }

  set phone(value: string) {
    this._phone = value;
  }
}

File: BE-Cinema/src/main/java/com/datn/cinema/dto/UserNoAccountDTO.java
package com.datn.cinema.dto;

public class UserNoAccountDTO {
    private String name;
    private String email;
    private String idCard;
    private String phone;

    public UserNoAccountDTO() {
    }

    public UserNoAccountDTO(String name, String email, String idCard, String phone) {
        this.name = name;
        this.email = email;
        this.idCard = idCard;
        this.phone = phone;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getIdCard() {
        return idCard;
    }

    public void setIdCard(String idCard) {
        this.idCard = idCard;
    }

    public String getPhone() {
        return phone;
    }

    public void setPhone(String phone) {
        this.phone = phone;
    }
}
